docroot/modules/custom/yukon_w3_custom/css/yukon_w3_custom.css

```diff
@@ -0,0 +1,58 @@
+ul.tabs--secondary.pagination.pagination-sm {
```

Comment:
Don't use css please.
Try to use tailwind as much as possible. For customizing use scss as done everywhere else on the site please